examples/pytorch/logging/pytorch_log_model.ipynb

```diff
@@ -0,0 +1,440 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "7066087c-6947-4a50-a7fb-6ab71c004ccf",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "# Log Your PyTorch Model to mlflow\n",
+    "\n",
+    "This guide will walk you through how to save your PyTorch model to mlflow and load the saved model for inference. Saving a pretrained/finetuned model in MLflow allows you to easily share the model or deploy it to production.\n",
+    "\n",
+    "We will cover how to:\n",
+    "- Define a simple pytorch model\n",
+    "- Set a model signature for our logged model to define inputs and outputs to the mlflow model\n",
+    "- Log our model to MLflow server\n",
+    "- Load the model back from storage to use in other notebooks"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Install dependencies"
```

Comment:
This is actually my bad, I just realized this notebook does not mention how to start the MLflow server or connecting to a hosted tracking server. Let's add a section before "installing dependencies", we can call it "Start MLflow Server", inside it we should mention users can either start their server by `mlflow ui` (not compatible with hosted notebooks), or using tracking server, and refer to this doc: https://mlflow.org/docs/latest/getting-started/tracking-server-overview/index.html